SDKBuildScripts/unity_build_beta.bat

```diff
@@ -0,0 +1,5 @@
+setlocal
```

Comment:
Don't commit this file.
Rename this file to "beta_unity.bat", so that it is filtered by the .gitignore, and that will be sufficient.

Reply:
Done !
Still getting used to how pull request works, the first two commits weren't really intended to be there. I'll try to be more carefull about what's inside a PR.